### DISCUSSION TITLE
Could not find devcontainercli-root error on codespace creation

### DISCUSSION THREAD

**Discussion by lucidenis** [community/community#76353] [Codespaces]
### Select Topic Area

Bug

### Body

Hi,

All our `docker-outside-of-docker` based codespaces started failing a couple days ago with the following error showing in `creation.log`:
```
ERROR: could not find /tmp/devcontainercli-root/container-features/0.52.1-1700494756266: stat /tmp/devcontainercli-root/container-features/0.52.1-1700494756266: no such file or directory
```

**Comment by jeff-rqdn** [comment]
I've been seeing the same error in my logs since friday for my team's python containers:
```
2023-11-17 22:21:45.501Z: ERROR: could not find /tmp/devcontainercli-root/container-features/0.52.1-1700182450886: stat /tmp/devcontainercli-root/container-features/0.52.1-1700182450886: no such file or directory
```

**Comment by adamstegman** [comment]
Replacing the `build` configuration for our main container in `docker-compose.yml` seems to avoid the issue. We can put the lines from our Dockerfile into our `updateContentCommand` instead for now.

**Reply by dmattia** [reply]
This worked for me as well, thanks

**Comment by kourylape** [comment]
We're seeing the same "could not find" error with ` /tmp/devcontainercli-root/container-features/0.52.1-XXX`.

We noticed that rebuilding the codespace immediately after creating works fine.

**Comment by adamstegman** [comment]
Checked on this again this week and it seems resolved now. I've reverted back to using `build`. 🤷
